recipes/libarchive/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+#
+source activate "${CONDA_DEFAULT_ENV}"
+
+# Needed for the tests.
+export CFLAGS="-std=c99 ${CFLAGS}"
+
+if [ "`uname`" == 'Darwin' ]
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+else
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+
+aclocal
+autoconf
+autoreconf -ivf
```

Comment:
Straight from source so we need to use `autoconf` to generate the `configure` file.